src/Core/CustomMethods.php

```diff
@@ -60,10 +60,10 @@ public function __call($method, $arguments)
         }
 
         switch (true) {
-            case isset($config['callback']): {
+            case isset($config['callback']):
                 return $config['callback']($this, $arguments);
-            }
-            case isset($config['property']) : {
+                break;
```

Comment:
Why does a return need a break?

Reply:
It’s a rule isn’t it? Every case must have a break?